README.md

```diff
@@ -203,45 +203,53 @@ struct Time : twig::stronk<Time, double, twig::unit>
     using stronk::stronk;
 };
 
-// Note: For the specializer macros you need to call them from within the twig namespace:
-namespace twig
+// Lets say you want to use a custom defined stronk type for certain unit combinations.
+// Lets introduce our own `Speed` type:
+struct Speed : twig::stronk<Speed, double, twig::can_hash, twig::divided_unit<Distance, Time>::skill>
```

Comment:
Consider removing `can_hash` from this example, if it is not directly related to this example. Otherwise it might just be more confusing